ledger/acctupdates.go

```diff
@@ -124,6 +134,20 @@ type modifiedResource struct {
 	ndeltas int
 }
 
+// A modifiedValue represents the value that has been modified since the
+// persistent state stored in the account DB (i.e., in the range of rounds
+// covered by the accountUpdates tracker).
+type modifiedValue struct {
+	// data stores the most recent value (nil == deleted)
+	data *string
+
+	// ndelta keeps track of how many times the key for this value appears in
+	// accountUpdates.deltas.  This is used to evict modifiedValue entries when
+	// all changes to a key have been reflected in the kv table, and no
+	// outstanding modifications remain.
+	ndeltas int
+}
```

Comment:
There have been (very welcome) objections to the name `modifiedValue`, it feels as though it conveys almost no meaning.  It follows the convention established by `modfiedAccount` and `modifiedResource` - it is the new value of a key/value per.  And just like those structures, it does have the "key", (they don't contain the address, or the creatable id).  So this is _just_ the "Value" of a "KeyValue" pair.  Anyway, it ends up reading funny.

Should I use `modifiedKeyValue` even though there is no Key in here?  `modfiedKValue` as a way to show something funny is going on, and there is no Key in here?

Reply:
@jannotti + reviewers If we're making a change, how does `modifiedKvValue` sound?

* The proposal deviates from the _modified_ naming convention in an attempt to demarcate its intended usage.
* If that doesn't seem palatable, I'd begrudgingly fall back to `modifiedKValue`.

Reply:
I liked modified KvValue. Or reads well if you think of Kv as an adjective. Go standard would maybe say it should be KVValue. But that is gross.

Reply:
I'm going go with `modifiedKvValue`, PR into branch to follow.